### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by ulilaube**
several small and independen fix split off from PR #342

### CODE REVIEW

File: CppCoreGuidelines.md [hunk 1]
```diff
@@ -10328,7 +10328,7 @@ In a class template, nonvirtual functions are only instantiated if they're used
 
 * Flag a class template that declares new (non-inherited) virtual functions.
 
-## <a name="SS-tpg-concepts"></a> TPG.concepts: Concept rules
+## <a name="SS-concepts"></a> T.concepts: Concept rules
```
Comment (gdr-at-ms) [inline CppCoreGuidelines.md]: Why the renaming from TGP to T?

**Comment by gdr-at-ms** [comment]
Could you squash the PR?

File: CppCoreGuidelines.md (hunk 1 above)
Comment (ulilaube) [inline CppCoreGuidelines.md]: consistency, it appears no where else

**Comment by ulilaube** [comment]
squashed

File: CppCoreGuidelines.md (hunk 1 above)
Comment (gdr-at-ms) [inline CppCoreGuidelines.md]: Ah yes, I guess the thing that confused me was that on line 1391 you changed the anchor to "SS-t-concepts"; but here it is just "SS-concepts" -- note the missing "-t".

Comment (ulilaube) [inline CppCoreGuidelines.md]: true but that needs some love as well,
some anchors are
SS-t-foobar
SS-temp-foobar
SS-foobar
in the chapter on templates

**Pull request merged**